lms/static/js/verify_student/views/image_input_view.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Allow users to upload an image using a file input.
+ *
+ * This uses HTML Media Capture so that iOS will
+ * allow users to use their camera instead of choosing
+ * a file.
+ */
+
+ var edx = edx || {};
+
+ (function( $, _, Backbone, gettext ) {
+    'use strict';
+
+    edx.verify_student = edx.verify_student || {};
+
+    edx.verify_student.ImageInputView = Backbone.View.extend({
+
+        template: "#image_input-tpl",
```

Comment:
Alternative technique is to wrap the element in the underscore template function
template: _.template( $('#image_input-tpl') )
then in render function can just call this.template();